clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -2494,6 +2494,19 @@ void DarwinClang::AddClangCXXStdlibIncludeArgs(
                    << "\"\n";
     }
 
+    // Check for the folder where the executable is located, if different.
+    if (getDriver().getInstalledDir() != getDriver().Dir) {
+      InstallBin = llvm::StringRef(getDriver().Dir.c_str());
```

Comment:
`StringRef` has a `const std::string&` constructor, so the `.c_str()` part is unnecessary.